keybot/main.go

```diff
@@ -40,6 +44,10 @@ func kingpinHandler(args []string) (string, error) {
 
 	cancel := build.Command("cancel", "Cancel any existing builds")
 
+	buildWindows := build.Command("windows", "start a windows build")
+	testWindows := buildTest.Arg("windows", "Start a windows test build").String()
+	cancelWindows := cancel.Arg("windows", "Cancel last windows build").String()
```

Comment:
Maybe have the build output the number... and then cancel takes a number as arg... then you don't need that global